Formula/traefik.rb

```diff
@@ -0,0 +1,61 @@
+require "language/go"
+
+class Traefik < Formula
+  desc "Træfik, a modern reverse proxy"
+  homepage "https://traefik.io/"
+  url "https://github.com/containous/traefik/releases/download/v1.3.8/traefik-v1.3.8.src.tar.gz"
+  sha256 "6fce36dd30bb5ae5f91e69f2950f22fe7a74b920e80c6b441a0721122f6a6174"
+  head "https://github.com/containous/traefik.git"
+
+  depends_on "go" => :build
+  depends_on "node" => :build
+  depends_on "yarn" => :build
+
+  go_resource "github.com/jteeuwen/go-bindata" do
+    url "https://github.com/jteeuwen/go-bindata.git",
+        :revision => "a0ff2567cfb70903282db057e799fd826784d41d"
+  end
+
+  def install
+    ENV["GOPATH"] = buildpath
+    dir = buildpath/"src/github.com/containous/traefik"
+    dir.install buildpath.children
```

Comment:
`(buildpath/"src/github.com/containous/traefik").install buildpath.children` is less than 80 character so we don't need the intermediate variable

Reply:
But this variable used below too.
And this is a usual occurrence in formulas. (search "dir = buildpath/")

Reply:
The general rule is not to introduce `dir.install` unless it goes over 80 here.

Reply:
But what about variable reuse?

Reply:
As a general rule of life: fewer lines = fewer bugs.

Reply:
Done.